Resolving Punctuation Errors in Coalition Jobs

MessyMix opened the pull request:
For reference:
http://www.dailywritingtips.com/punctuation-mistakes-1-unnecessary-commas/

Removing unnecessary commas:

Commit 1: "I, need food" would be incorrect, it is "I need food".

Commit 2: "I need a car, to get food" should be "I need a car to get food". The extra comma is extraneous.

Commit 3: "As part of a retirement plan, transport these 500 retirees" is incorrect. (https://owl.english.purdue.edu/owl/resource/597/01/) The phrase "as part of the retirement plan" does not apply to you, who is transporting the retirees. Instead, it should be "As part of a retirement plan, these 500 retirees need transportation". It's a grammatical error otherwise.

Commit 4: Same as above. The Arach interspecies are done correctly, so I just changed everything to fit the Arach.

Commit 5: "I need a car, by tuesday" --> "I need a car by tuesday".

Bladewood commented:
I'm not sure about the Interspecies missions, but everything else seems good (with your commits).

MessyMix commented:
@Bladewood it's a complex grammar error, but if you look at Arach interspecies, it is done the way that I fixed the others, and I know that the Arach ones are grammatically correct, so I changed the others too. It's also just a matter of consistency.

Bladewood commented:
screw consistency, we've got three different alien species here, we need to make them unique
/joke

MessyMix commented:
xD, but then the other two species that are currently identical(ly wrong)
need variety! Let's make them all more wrong! :DDD

On 26 February 2017 at 12:36, Bladewood <notifications@github.com> wrote:

> screw consistency, we've got three different alien species here, we need
> to make them unique
> /joke
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/endless-sky/endless-sky/pull/2193#issuecomment-282572360>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATgLHECYHTyZX_R1fba6Kzwh-NFCcr9Rks5rgbgUgaJpZM4MMclh>
> .
>

endless-sky commented:
`<destination>` is replaced with "`<planet>`, in the `<system>` system." So, it always has to end with either a period or a comma, because "in the `<system>` system" is a parenthetical phrase.

For example, "Earth, in the Sol system, needs...", not "Earth, in the Sol system needs..."

MessyMix commented:
@endless-sky, 
![image](https://cloud.githubusercontent.com/assets/20450076/23342054/07277790-fc21-11e6-8f4a-4e53f9f16711.png)

endless-sky commented:
Oops. That probably shows up in a lot of other places too, then. Somehow I had it wrong in the "creating missions" wiki page and I guess I never noticed that that wasn't how the code was working.

endless-sky commented:
Fixed quite a few other cases of that in 1767200. Thanks for catching that.

MessyMix commented:
😉 No problem.
